pkg/server/server.go

```diff
@@ -318,6 +320,10 @@ func (s *Server) ListObjects(ctx context.Context, req *openfgav1.ListObjectsRequ
 	))
 	defer span.End()
 
+	if err := req.Validate(); err != nil {
+		return nil, status.Error(codes.InvalidArgument, err.Error())
+	}
```

Comment:
Do you mind sharing a screenshot showing that the http status code of a validation error hasn't changed?

Reply:
Yeah I'll get one here in a sec, but the fact none of the tests failed should be a good indicator no?

Reply:
And the HTTP status code mapping still happens in the grpc-gateway middleware, which applies to the standalone server. So as long as the `codes.InvalidArgument` is the same as it was before we gucci.

Reply:
Here's a sample screenshot. 
<img width="1305" alt="Screenshot 2023-08-30 at 1 52 26 PM" src="https://github.com/openfga/openfga/assets/2899204/2cc1bd79-4433-446b-8281-62a743b85576">

The HTTP status code is 400 👍